tools/db_bench_tool.cc

```diff
@@ -8185,6 +8207,50 @@ class Benchmark {
     }
   }
 
+  void Backup(ThreadState* thread) {
+    DB* db = SelectDB(thread);
+    std::string backup_dir = FLAGS_db + "/backups";
+    std::unique_ptr<BackupEngineOptions> engine_options_(
+        new BackupEngineOptions(backup_dir));
+    Status s;
+    BackupEngine* backup_engine;
+    if (FLAGS_backup_rate_limit > 0) {
+      engine_options_.get()->backup_rate_limiter.reset(NewGenericRateLimiter(
```

Comment:
Is there a particular reason why we need the raw pointer of engine_options_ `engine_options_.get()` but not directly with unique_ptr?

Reply:
Good catch, no particular reason: IDE being weird and me being rusty in C++ ..

Reply:
Updated to use unique_ptr directly.